website/docs/r/ec2_client_vpn_network_association.html.markdown

```diff
@@ -26,12 +27,14 @@ The following arguments are supported:
 
 * `client_vpn_endpoint_id` - (Required) The ID of the Client VPN endpoint.
 * `subnet_id` - (Required) The ID of the subnet to associate with the Client VPN endpoint.
+* `security_groups` - (Optional) A list of up to five security groups to apply to the target network.
+* `vpc_id` - (Optional) The ID of the VPC in which the target network is locatated.  This is a required argument for assigning security groups to a network association.
```

Comment:
Typo 😎 `located` although if we infer it from the subnet as mentioned above, this line can be removed.